diff --git a/node_modules/@nrwl/expo/plugins/index.d.ts b/node_modules/@nrwl/expo/plugins/index.d.ts
new file mode 100644
index 0000000..28c064b
--- /dev/null
+++ b/node_modules/@nrwl/expo/plugins/index.d.ts
@@ -0,0 +1 @@
+export { withNxMetro } from './with-nx-metro';
diff --git a/node_modules/@nrwl/expo/plugins/index.js b/node_modules/@nrwl/expo/plugins/index.js
new file mode 100644
index 0000000..bcf1c1b
--- /dev/null
+++ b/node_modules/@nrwl/expo/plugins/index.js
@@ -0,0 +1,6 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.withNxMetro = void 0;
+var with_nx_metro_1 = require("./with-nx-metro");
+Object.defineProperty(exports, "withNxMetro", { enumerable: true, get: function () { return with_nx_metro_1.withNxMetro; } });
+//# sourceMappingURL=index.js.map
\ No newline at end of file
